packages/analytics/ios/RNFBAnalytics/RNFBAnalyticsModule.m

```diff
@@ -142,6 +142,22 @@ - (dispatch_queue_t)methodQueue {
       return resolve([FIRAnalytics appInstanceID]);
   }
 
+  RCT_EXPORT_METHOD(setDefaultEventParameters:
+    params:
```

Comment:
And I believe the API definition should look like this - I am able to commit these, then CI chews through...
```suggestion
```